lib/mobile_robot_simulator/vectorfield/points.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# Copyright (c) 2011 Autonomous Robots and Cognitive Systems Laboratory,
```

Comment:
Sometimes this reads Autonomous Robots and Cognitive Systems Laboratory, Universidad de Costa Rica and others Autonomous Robots and Cognitive Systems Laboratory; without the Universidad de Costa Rica. Standarize to one or the other please.